lua/core/menu/settings.lua

```diff
@@ -0,0 +1,94 @@
+local textentry= require 'textentry'
+
+local m = {
+  pos = 1,
+  list = {"RESET", "PASSWORD >", "BATTERY WARNING"},
+  pages = {"RESET", "PASSWORD"}
+}
+
+m.key = function(n,z)
+  if n==2 and z==1 then
+    _menu.set_page("HOME")
```

Comment:
for consistency with the other `SYSTEM` sub-pages, should this be `_menu.set_page("SYSTEM")` ? otherwise, K2 jumps back two menu layers, instead of one.